recipes/root/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.13)
+project(PackageTest CXX)
+
+find_package(ROOT REQUIRED)
+
+include_directories(${CMAKE_CURRENT_SOURCE_DIR})
+ROOT_GENERATE_DICTIONARY(G__Event Event.hpp OPTIONS -inlineInputHeader LINKDEF EventLinkDef.h )
+
+add_library(Event SHARED Event.cpp G__Event.cxx)
+target_link_libraries(Event ROOT::ROOT)
+
+add_executable(testrootdictionaries testrootdictionaries.cpp)
+target_link_libraries(testrootdictionaries Event ROOT::ROOT)
```

Should fix the error

```suggestion
target_link_libraries(Event ROOT::ROOT)
set_property(TARGET Event PROPERTY CXX_STANDARD 11)

add_executable(testrootdictionaries testrootdictionaries.cpp)
target_link_libraries(testrootdictionaries Event ROOT::ROOT)
set_property(TARGET testrootdictionaries PROPERTY CXX_STANDARD 11)
```

---

Thanks for this suggestion. This will fix this error but might create others. 

If ROOT is compiled with C++ >= 17 I have had problems in the past compiling C++11 applications against ROOT (due to things like `std::string_view` in the ROOT headers).

In 66718b6 the `test_package` uses the same `CMAKE_CXX_STANDARD` that ROOT was built with. This should fix the failing clang5 build.